### PR TITLE
[vcpkg docs] Update ko_KR translation

### PR DESCRIPTION
**Pull request by jnooree**
**Describe the pull request**

- #### What does your PR fix?  
  Updates ko_KR translation.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  Unrelated.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Unrelated.

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Unrelated.

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by PhoebeHui** [comment]
cc @AkiaCode, @vyujing, could you help review?

File: README_ko_KR.md [hunk 1]
```diff
@@ -305,14 +321,26 @@ Vcpkg는 오픈소스 프로젝트입니다, 따라서 여러분의 기여로 
...
+라이브러리는 각 라이브러리의 원저자가 설정한 라이선스에 따라 제공됩니다. 가능한 경우, vcpkg는
+`installed/<triplet>/share/<port>/copyright`에 관련된 라이선스를 저장합니다.
+
+# 보안
+
+Vcpk가 제공하는 대부분의 port는 각각의 라이브러리를 빌드할 때
+원 개발자들이 권장하는 빌드 시스템을 이용하고,
+소스 코드와 빌드 도구를 각각의 공식 배포처로부터 다운로드합니다.
+방화벽 뒤에서 사용하는 경우, 어떤 port를 설치하느냐에 따라 필요한 접근 권한이 달라질 수 있습니다.
+만약 "air gapped" (역자 주: 인터넷 연결이 불가능한) 환경에서 설치해야만 한다면,
```
Comment (AkiaCode) [inline README_ko_KR.md:334]: `역자 주` 부분을 없애는 게 좋은 거 같습니다.

**Review by AkiaCode — APPROVED** [review]
Thanks for @jnooree's contribution!

**Review by PhoebeHui — APPROVED** [review]
Thanks @AkiaCode for your review!

**Pull request merged**